@here/harp-datasource-protocol/lib/StyleSetEvaluator.ts

```diff
@@ -912,6 +918,11 @@ export class StyleSetEvaluator {
         if (style._usesFeatureState !== undefined) {
             technique._usesFeatureState = style._usesFeatureState;
         }
+        setTechniqueRenderOrderOrPriority(
```

Comment:
This is basically the main part of this change, the RenderOrder from the priorities is now already applied in the StyleSetEvaluator, instead of later on during geometry creation.

Reply:
all the other changes are basically needed, to pass through the priorities to the StyleSetEvaluator.